content/docs/introduction/glossary.md

```diff
@@ -108,6 +108,12 @@ write into a format the other system can understand.
 
 A remote write endpoint is what Prometheus talks to when doing a remote write.
 
+### Sample
+
+A sample is a single value at a point in time in a time series.
+
+In Prometheus, each sample consists of a float64 representing some metric and a millisecond-precision timestamp.
```

Comment:
"some metric" doesn't make sense here, as a sample is part of a time series is part of a metric so this is confusingly circular.